server/sonar-server/src/main/java/org/sonar/server/es/DefaultIndexSettings.java

```diff
@@ -45,6 +45,8 @@
   public static final String INDEX = "index";
   public static final String ANALYZED = "analyzed";
   public static final String STRING = "string";
+  public static final String FIELD_TYPE_TEXT = "string";  // ES 5: change to text
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)